docs/REST API/rest-api.md

```diff
@@ -6936,12 +6982,98 @@ Deletes the session specified by ID. When the user associated with the session n
 
 ## Software
 
+- [Add software](#add-software)
+- [Download software](#download-software)
+- [Delete software](#delete-software)
 - [List software titles](#list-software-titles)
 - [List software versions](#list-software-versions)
 - [Count software](#count-software)
 - [Get software title](#get-software-title)
 - [Get software version](#get-software-version)
 
+### Add software
+
+Add software to install on macOS and Windows hosts during enrollment.
+
+`POST /api/v1/fleet/software/manage`
```

Comment:
I know we've gone back and forth on this but I think we be brave and cut `/manage` so that it's just `POST /software`.

What do you think?